### PR TITLE
OBSDOCS-722 - Logging 5.6.15 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.11. 4.12, 4.13

Issue: [OBSDOCS-722](https://issues.redhat.com//browse/OBSDOCS-722)

Link to docs preview: https://70383--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/logging_release_notes/logging-5-6-release-notes

QE review: @anpingli  / @QiaolingTang / @kabirbhartiRH 
- [x] QE has approved this change.

Additional information:  CPs to be created after review.

### CODE REVIEW

**Comment by libander** [comment]
SME review please: @periklis / @jcantrill / @xperimental / @cahartma

File: modules/logging-release-notes-5-6-15.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//module included in logging-5-6-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-6-15"]
+= Logging 5.6.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:0270[OpenShift Logging Bug Fix Release 5.6.15].
+
+[id="logging-release-notes-5-6-15-known-issue"]
+== Known issue
+Currently, the log alert is not available on an IPv6-enabled cluster. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/LOG-4892[LOG-4892])
```
Comment (periklis) [inline modules/logging-release-notes-5-6-15.adoc:9]: As mentioned elsewhere, the new text is available on https://issues.redhat.com/browse/LOG-4892

**Review by periklis — CHANGES_REQUESTED** [review]

**Review by periklis — COMMENTED** [review]
LGTM

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Comment by skrthomas** [comment]
I saw your reply on my other bug fix comment, and I totally get ya. Fine by me to keep it consistent w/ other titles.

**Pull request merged**